NuGet.config

```diff
@@ -11,6 +11,7 @@
     <add key="vs-buildservices" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-buildservices/nuget/v3/index.json" />
     <add key="xamarin-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/xamarin-impl/nuget/v3/index.json" />
     <add key="maui-nugets" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json" />
+     <add key="nuget" value="https://api.nuget.org/v3/index.json" />
```

Comment:
@Sweekriti91 I'm pretty sure we can't do this. What package are you needing here?

Reply:
Ohhhh that's was for maui-check. Okay let me remove maui-check and I'll add the dotnet commands the maui-check does under the hood instead. I'll remove this.

Reply:
You can get maui-check added to one of the public azdo feeds you have listed and remove nuget.org as a source.